src/Pages/CartPage/CartComponents/CartProduct.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { addProductQty, reduceProductQty, removeFromCart, addToWishlist, findProductInList } from '../../../Util/index';
+import { useCart, useWishlist } from '../../../Contexts/index'
+
+const CartProduct = ({ cartItem }) => {
+    const { title, imgSrc, altTxt, subtitle, qty, reviewsnum, currentprice, discount, originalprice, rating, shortdesc } = cartItem
+    const { cartDispatch } = useCart()
+    const { wishlistState, wishlistDispatch } = useWishlist()
+
+    const updateCartList = async (product, getUpdatedCart) => {
+        const updatedCart = await getUpdatedCart(product)
+        cartDispatch({ type: 'UPDATE_CART', payload: updatedCart })
+    }
+
+    const moveFromCart = async (cartProduct) => {
+        const newWishlist = []
+        const updatedCart = await removeFromCart(cartProduct)
+        cartDispatch({ type: 'UPDATE_CART', payload: updatedCart })
+        const foundId = findProductInList(cartProduct._id, wishlistState)
+        if (!foundId) {
+            newWishlist = await addToWishlist(cartProduct)
+        } else {
+            console.log("product already in cart. update qty in cart page.")
+            newWishlist = await wishlistState
+        }
```

Comment:
changes: I don't think "await" is required in line 24, because it is not an API call.